packages/tpetra/core/src/Tpetra_Core.cpp

```diff
@@ -242,10 +242,10 @@ namespace Tpetra {
 #endif // defined(HAVE_TPETRACORE_MPI)
       initKokkosIfNeeded (argc, argv, myRank);
 
-      // Add Kokkos::deep_copy() to the TimeMonitor if the environment says so
+      // Add Kokkos calls to the TimeMonitor if the environment says so
       Tpetra::Details::AddKokkosDeepCopyToTimeMonitor();
-      // Add Kokkos::fence() to the TimeMonitor if the environment says so
       Tpetra::Details::AddKokkosFenceToTimeMonitor();
+      Tpetra::Details::AddKokkosKernelsToTimeMonitor();
```

Comment:
This adds kernels written using Kokkos, not "Kokkos Kernels", right? Maybe just call it `AddKernelsToTimeMonitor`?